### ISSUE TITLE
adblock - dependency on logger

### ISSUE THREAD

**Issue by ratsputin** [openwrt/packages#10438]
Maintainer: @dibdot 
Environment: OpenWrt SNAPSHOT r11372-2d00cf7515 / LuCI Master git-19.305.42300-af5dc26

Description:

An (apparently) relatively recent update to `adblock.sh` leverages `logger` command in the `f_log()` routine.  Unfortunately, in deployments where a standard syslog has been implemented, `logger` is unavailable and adblock fails to start.

```
root@orion:/usr/var/log# adblock.sh
/usr/bin/adblock.sh: line 919: logger: not found
```

**Comment by ratsputin** [comment]
It appears I've misinterpreted the nature of the problem.  Adblock does have a dependency on `logger`; however, the build system apparently doesn't reflect this dependency.  I am adding the `logger` package to my build to see if it resolves the issue.

**Comment by ratsputin** [comment]
This is definitely the issue.  Please add a dependency on `logger` for `adblock` in the build system.

**Comment by dibdot** [comment]
Will be fixed with #10446 (made logger optional)
